src/libotutil/ot-fs-utils.c

```diff
@@ -277,3 +277,26 @@ ot_get_dir_size (int dfd, const char *path, guint64 blocksize, guint64 *out_size
 
   return TRUE;
 }
+
+/* Check whether a path exists */
+gboolean
+ot_path_exists (const char *path, gboolean *out_val, GError **error)
```

Comment:
I'm fine with this but note there is a lot of other usage of `glnx_fstatat_allow_noent` in the code for a similar pattern.